src/runtime/schema/schema.ts

```diff
@@ -149,3 +152,40 @@ export function create(appState: AppState): SchemaInternal {
 function emptyExceptionMessage() {
   return `Your GraphQL schema is empty. This is normal if you have not defined any GraphQL types yet. If you did however, check that your files are contained in the same directory specified in the \`rootDir\` property of your tsconfig.json file.`
 }
+
+function assertNoMissingTypesDev(
+  schema: NexusSchema.core.NexusGraphQLSchema,
+  missingTypes: Index<NexusSchema.core.MissingType>
+) {
+  const missingTypesNames = Object.keys(missingTypes)
+
+  if (missingTypesNames.length === 0) {
+    return
+  }
+
+  const schemaTypeMap = schema.getTypeMap()
+  const schemaTypeNames = Object.keys(schemaTypeMap).filter(
+    (typeName) => !NexusSchema.core.isUnknownType(schemaTypeMap[typeName])
+  )
+
+  if (DevMode.isDevMode()) {
+    missingTypesNames.map((typeName) => {
+      const suggestions = NexusSchema.core.suggestionList(typeName, schemaTypeNames)
+
+      let suggestionsString = ''
+
+      if (suggestions.length > 0) {
+        suggestionsString = ` Did you mean ${suggestions
+          .map((s) => `"${chalk.greenBright(s)}"`)
+          .join(', ')} ?`
+      }
+
+      log.error(`Missing type "${chalk.greenBright(typeName)}" in your GraphQL Schema.${suggestionsString}`)
+    })
+  } else {
+    Process.fatal(
```

Comment:
Throw?

Reply:
Why would we want to throw instead of a clean fatal? 🤔